src/sentry/static/sentry/app/actionCreators/modal.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import {css} from 'react-emotion';
 
-import ModalActions from 'app/actions/modalActions';
+import ModalActions, {ModalOptions} from 'app/actions/modalActions';
 
 /**
  * Show a modal
  */
-export function openModal(renderer, options) {
+export function openModal(
+  renderer: (mod: any) => React.ReactNode,
```

Comment:
I think we can do better than `any` here.

These are the `RenderProps` no? Probably should be `Partial` too.

Reply:
@EvanPurkhiser Why would we use `Partial` here? I think we only call the `renderer` once and that's in render function of `GlobalModal` where we call `children`.

Reply:
ah you're right sorry it wouldn't be partial. But all modal components will have to have partial of the render props, so that they can accept them.